pkg/providers/aws/provider_postgres.go

```diff
@@ -168,11 +168,11 @@ func (p *PostgresProvider) ReconcilePostgres(ctx context.Context, pg *v1alpha1.P
 		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 	}
 
-	//networkManager isEnabled checks for the presence of valid RHMI subnets in the cluster vpc
-	//when rhmi subnets are present in a cluster vpc it indicates that the vpc configuration
+	//networkManager isEnabled checks for the presence of valid subnets in the cluster vpc
+	//when subnets are present in a cluster vpc it indicates that the vpc configuration
 	//was created in a cluster with a cluster version <= 4.4.5
 	//
-	//when rhmi subnets are absent in a cluster vpc it indicates that the vpc configuration has not been created
+	//when subnets are absent in a cluster vpc it indicates that the vpc configuration has not been created
```

Comment:
this comment no longer makes sense, think just substitute rhoam for rhmi

Reply:
it refers to the cluster subnets so product installation shouldn't matter.